core/eolearn/core/visualizations.py

```diff
@@ -0,0 +1,547 @@
+        """ Plots shapes for one timestamp from geopandas GeoDataFRame
+
+        :param data_gpd: data to plot
+        :type data_gpd: geopandas.GeoDataFrame
+        :param timestamp: timestamp to plot data for
+        :type timestamp: datetime
+        :param epsg_number: in which crs is the data to plot
+        :type epsg_number: int
+        :return: visualization
+        :rtype: geoviews
+        """
+        out = data_gpd.loc[data_gpd[self.timestamp_column] == timestamp]
+        return gv.Polygons(out, crs=ccrs.epsg(epsg_number))
+
+    def plot_vector_timeless(self, feature_name):
+        """ Plot FeatureType.VECTOR_TIMELESS data
+
+        :param feature_name: name of the eopatch featrue
+        :type feature_name: str
+        :return: visalization
+        :rtype: geoviews
+        """
+        epsg_number = self.eopatch.bbox.crs.ogc_string().split(':')[1]
+        if epsg_number == 4326:
+            epsg_number = 3857
+            data_gpd = self.eopatch[FeatureType.VECTOR_TIMELESS][feature_name].to_crs({'init': 'epsg:3857'})
+        else:
+            data_gpd = self.eopatch[FeatureType.VECTOR_TIMELESS][feature_name]
+        return gv.Polygons(data_gpd, crs=ccrs.epsg(epsg_number), vdims=self.vdims)
+
+    def eopatch_da_to_rgb(self, eopatch_da, feature_name, crs):
+        """ Creates new xarray DataArray (from old one) to plot rgb image with hv.Holomap
+
+        :param eopatch_da: eopatch DataArray
+        :type eopatch_da: DataArray
+        :param feature_name: name of the feature to plot
+        :type feature_name:  str
+        :param crs: in which crs are the data
+        :type crs: sentinelhub.constants.crs
+        :return: eopatch DataArray with proper coordinates, dimensions, crs
+        :rtype: xarray.DataArray
+        """
+        timestamps = eopatch_da.coords['time'].values
+        bands = eopatch_da[..., self.rgb] * self.rgb_factor
+        bands = bands.rename({feature_name.replace('-', '_') + '_dim': 'band'}).transpose('time', 'band', 'y', 'x')
+        x_values, y_values = new_coordinates(eopatch_da, crs, CRS(3857))
+        eopatch_rgb = xr.DataArray(data=np.clip(bands.data, 0, 1),
+                                   coords={'time': timestamps,
+                                           'band': self.rgb,
+                                           'y': np.flip(y_values),
+                                           'x': x_values},
+                                   dims=('time', 'band', 'y', 'x'))
+        return eopatch_rgb
+
+    def plot_pixel(self, feature_type, feature_name):
+        """
+        Plots one pixel through time
+        :return:
+        :rtype:
+        """
+        data_da = array_to_dataframe(self.eopatch, (feature_type, feature_name))
+        if self.mask:
+            data_da = self.mask_data(data_da)
+        if data_da.dtype == np.bool:
+            data_da = data_da.astype(np.int8)
+        return data_da.hvplot(x='time')
+
+    def mask_data(self, data_da):
```

Comment:
I am thinking about moving [code from this method](https://github.com/sentinel-hub/eo-learn/blob/develop/features/eolearn/features/interpolation.py#L92) into a utility function of `eolearn.core`. It is a general masking code which handles multiple combinations data and mask features types. Then it could also be used here.

Reply:
I put it on my TODO list, we can leave as it is for this PR.